src/pages/models/[id].tsx

```diff
@@ -18,6 +18,7 @@ interface Params extends ParsedUrlQuery {
 interface Props {
     modelId: ModelId;
     modelData: Model;
+    userIds: UserId[];
```

Comment:
I don't think that we should include data that is only used in edit mode in the static page data. This just needlessly bloats the page size for normal viewers.

I think we should use the webApi to fetch data needed for edit mode.

Reply:
Makes sense.